domain/model/service/modelservice.go

```diff
@@ -157,3 +212,23 @@ func (s *ModelService) GetStatus(ctx context.Context) (model.StatusInfo, error)
 		Since:  now,
 	}, nil
 }
+
+// GetEnvironProviderVersion retrieves the version of the environment provider associated with the model.
+//
+// Errors:
+// - [modelerrors.NotFound]: Returned if the model does not exist.
```

Comment:
We current use `[error]: When ....`

NB We will be transitioning to lower case after the colon but let's stay consistent for now.

Reply:
Noted with thanks Ian!